### PR TITLE
section 1, 2 and 3 completed

### PR DESCRIPTION
**Pull request by kavicastelo**

### CODE REVIEW

File: client/src/app/components/about/about.component.html [hunk 1]
```diff
@@ -126,5 +126,84 @@ <h4>Contacts</h4>
         </div>
       </div>
     </div>
+
+    <div class="main-banner">
```
Comment (kavicastelo) [inline client/src/app/components/about/about.component.html:130]: main banner element

File: client/src/app/components/about/about.component.html [hunk 2]
```diff
@@ -126,5 +126,84 @@ <h4>Contacts</h4>
...
+        <div class="d-sm-flex align-items-center justify-content-between">
+          <div class="title-1">About Us</div>
+          <div class="d-flex align-items-center" style="height: auto!important;">
+            <span>Home</span>&nbsp; > &nbsp;<pre routerLink="/about">About Us</pre>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="sec-2">
```
Comment (kavicastelo) [inline client/src/app/components/about/about.component.html:141]: section 2 same section in home page

File: client/src/app/components/about/about.component.html [hunk 3]
```diff
@@ -126,5 +126,84 @@ <h4>Contacts</h4>
...
+                   style="background-color: #e27614; width: 100%; height: calc(100vh - 80px); transform: skewY(-10deg); position: absolute; bottom: -10px; left: -30px; z-index: 0">
+              </div>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="sec-3">
```
Comment (kavicastelo) [inline client/src/app/components/about/about.component.html:186]: section 3 same section in home page

File: client/src/app/components/about/about.component.ts [hunk 4]
```diff
@@ -9,9 +11,48 @@ import {Component, ViewEncapsulation} from '@angular/core';
 export class AboutComponent {
 
   isMenuOpen: boolean = false;
+  counter: number = 0;
+  sec2counter: number = 300;
+
+  constructor(private sanitizer: DomSanitizer, private cdr: ChangeDetectorRef, private elementRef: ElementRef, private renderer: Renderer2) {
```
Comment (kavicastelo) [inline client/src/app/components/about/about.component.ts:17]: unwanted imports

**Review by kavicastelo — COMMENTED** [review]
by kavi

File: client/src/app/components/about/about.component.html (hunk 2 above)
Comment (kavindu0) [reply client/src/app/components/about/about.component.html:141]: reviewed

File: client/src/app/components/about/about.component.html (hunk 1 above)
Comment (kavindu0) [reply client/src/app/components/about/about.component.html:130]: reviewed

File: client/src/app/components/about/about.component.html (hunk 3 above)
Comment (kavindu0) [reply client/src/app/components/about/about.component.html:186]: reviewed

File: client/src/app/components/about/about.component.ts (hunk 4 above)
Comment (kavindu0) [reply client/src/app/components/about/about.component.ts:17]: remove these unwanted imports

Comment (kavindu0) [inline client/src/app/components/about/about.component.ts:17]: @kavicastelo fix issue

**Review by kavindu0 — COMMENTED** [review]
by kavindu0

**Pull request merged**